dashboard/pages/hit_validation/stages/s1_screening_data_input.py

```diff
@@ -0,0 +1,106 @@
+from dash import html, dcc
+
+PARAMS_CONTAINER = html.Div(
+    children=[
+        html.H5(
+            children="Determination Parameters",
+        ),
+        html.Div(
+            children=[
+                html.Div(
+                    children=[
+                        html.Label(
+                            children="Concentration lower bound (µM)",
+                            className="form-label",
+                        ),
+                        dcc.Input(
+                            id="concentration-lower-bound-input",
+                            type="number",
+                            value=0,
+                            min=0,
+                            step=0.0001,
```

Comment:
Should we define the minimum and the step?

Reply:
step might not be necessary so I'll remove it,
but as for the minimum - I think that the concentration can't be lower than 0μM